content/en/dora_metrics/_index.md

```diff
@@ -1,37 +1,34 @@
 ---
 title: DORA Metrics
 kind: documentation
-description: Learn how to use DORA metrics to improve and measure software development.
+description: Learn how to use DORA metrics to measure and improve software development.
 aliases:
 - /continuous_integration/dora_metrics
 is_beta: true
 further_reading:
 - link: "https://app.datadoghq.com/release-notes?category=Software%20Delivery"
   tag: "Release Notes"
   text: "Check out the latest Software Delivery releases! (App login required)"
-- link: "/continuous_integration/tests"
-  tag: "Documentation"
-  text: "Learn about Test Visibility"
 - link: "/continuous_integration/pipelines"
   tag: "Documentation"
   text: "Learn about Pipeline Visibility"
+- link: "/continuous_delivery/deployments"
+  tag: "Documentation"
+  text: "Learn about Deployment Visibility"
+- link: "/continuous_integration/tests"
+  tag: "Documentation"
+  text: "Learn about Test Visibility"
 - link: "/code_analysis"
   tag: "Documentation"
   text: "Learn about Code Analysis"
-- link: "/tracing/service_catalog"
-  tag: "Documentation"
-  text: "Learn about the Service Catalog"
-- link: "https://github.com/DataDog/datadog-ci"
-  tag: "Source Code"
-  text: "Learn about the datadog-ci CLI tool"
 ---
 
 {{< site-region region="gov" >}}
 <div class="alert alert-warning">DORA Metrics is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
 {{< /site-region >}}
 
 {{< callout url="https://forms.gle/Eqq6uXfGjYxmqpjDA" header="false" >}}
-DORA Metrics are in private beta. Fill out the form below to be added to the waitlist.
+DORA Metrics private beta is closed. Fill out the form below to be added to the waitlist.
```

Comment:
NIT: If you click on the link, the form title still says `Private Beta Request`

Reply:
We'll change this :) Thanks for noticing this!